lib/browser-runtime.ts

```diff
@@ -18,7 +18,7 @@ import DataTable from "./data_table";
 
 import {
   assignRegistry,
-  freeRegistry,
+  freeRegistry, getRegistry,
```

Comment:
You should run all tests, this will reveal this as a problem (unused variable and incorrect formatting).

```
$ npm run test
```

Reply:
Got it, will remove this